source/Ship.cpp

```diff
@@ -425,6 +425,18 @@ void Ship::Load(const DataNode &node)
 			int count = (child.Size() >= 3) ? child.Value(2) : 1;
 			finalExplosions[GameData::Effects().Get(child.Token(1))] += count;
 		}
+		else if(key == "hit effect")
+		{
+			int count = (child.Size() >= 3) ? child.Value(2) : 1;
+			hullHitEffects[GameData::Effects().Get(child.Token(1))] += count;
+			hasHitEffect = true;
```

Comment:
How about this is called hull hit effect, as it normally is defined for the hull? Not something really important though

Reply:
I believe it was named this way because if shield effect isnt defined this one will be used for shields taking damage as well.

Reply:
Yh, I used "hit effect" for hull and default for shields, and a separate "shield hit effect" for just shields.

Reply:
This section has been touched upon with Quyykk, resolving this conversation for now.